### DISCUSSION TITLE
video download helper

### DISCUSSION THREAD

**Discussion by notimportant555** [aclap-dev/video-downloadhelper#841] [Help and Q&A]
### Your question

I was using download help with out a problem then it got updated with out my permission and now it will not work.....I have been using windows 7 all this time with no problem and not afford windows 10 so windows 7 is fine for me.....I have no clue what this beta is and went to a previous version (have no clue which 1) but had the look I had but just stays on queuing and  doesn't download....tried to ask jerome for help but he can't be bothered and can not find a downloader that works with a site I was using download helper for....man this is aggravating.....also not a technology person so I am sure I will not understand anyone

### Addon & CoApp version and details

I was using download help with out a problem then it got updated with out my permission and now it will not work.....I have been using windows 7 all this time with no problem and not afford windows 10 so windows 7 is fine for me.....I have no clue what this beta is and went to a previous version (have no clue which 1) but had the look I had but just stays on queuing and  doesn't download....tried to ask jerome for help but he can't be bothered and can not find a downloader that works with a site I was using download helper for....man this is aggravating.....also not a technology person so I am sure I will not understand anyone

**Comment by Mattmjs** [comment]
Read this if you want help
https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#start-a-new-qa-post

**Comment by notimportant555** [comment]
sorry do not understand the link    I am not a technology dictionary so I gave my issue and I already knew no one would understand so I gave up on your worthless site      bye
